docs/USAGE-GUIDE.md

```diff
@@ -669,6 +669,18 @@ callback function takes three arguments: a pointer to abitrary data for use
 within the callback, a pointer to a key which can be used to delete the
 cached entry, and a 64 bit unsigned integer specifying the size of this key.
 
+### s2n\_config\_set\_max\_fragment\_length
+
+```c
+int s2n_config_set_max_fragment_length(struct s2n_config *config, uint8_t mfl_code);
+```
+
+**s2n_config_set_max_fragment_length** allows the caller to set a TLS Maximum
+Fragment Length extension that will be used to fragment outgoing messages.
+s2n currently does not reject fragments larger than the configured maximum when
```

Comment:
I think you canmove `s2n currently does not reject fragments larger than the configured maximum when` over to your new opt-in API call docs